macros/parsers/parserMultiAnswer.pl

```diff
@@ -328,13 +332,18 @@ sub single_check {
 			. '</TABLE>';
 	}
 	if ($nonblank) {
-		$ans->{preview_latex_string} =
-			(
-				defined($self->{tex_format})
-				? sprintf($self->{tex_format}, @latex)
-				: join($self->{tex_separator}, @latex));
-		$ans->{preview_text_string} =
-			(defined($self->{format}) ? sprintf($self->{format}, @text) : join($self->{separator}, @text));
+		if ($hide_preview) {
```

Comment:
Why can't this just be `if ($ans->{scaffold_past_open}) {`, with the part above deleted?  (Although I don't think any of this is working anyway.)